pkg/apis/resource/v1alpha1/git/git_resource.go

```diff
@@ -195,6 +197,9 @@ func (s *Resource) GetInputTaskModifier(_ *v1beta1.TaskSpec, path string) (v1bet
 			WorkingDir: pipeline.WorkspaceDir,
 			// This is used to populate the ResourceResult status.
 			Env: env,
+			SecurityContext: &corev1.SecurityContext{
+				RunAsUser: &runAsUser,
```

Comment:
Mostly FYI, but we created this package because we were sick of declaring variables for these: https://github.com/knative/pkg/blob/0a1ec2e0d46c7e4ca0d0888510b1663d22a1bed7/ptr/ptr.go#L27-L31